README.md

```diff
@@ -37,6 +37,13 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd .
   - `brew install redis`
   - Follow post-install instructions to enable Redis on startup. Otherwise,
     launch it manually with `brew services start redis`.
+1. Install ClamAV
+  - `brew install clamav`
+  - `cd` into the directory listed by the post-install instructions (and replace '${post_install_dir}' with it below), then:
+    - `touch clamd.sock`
+    - `echo "LocalSocket ${post_install_dir}" > clamd.conf` 
+    - `echo "DatabaseMirror database.clamav.net" > freshclam.conf`
+    - `freshclam -v`
```

Comment:
I do not have those files in `/Users/vhaisaryanw/.brew/Cellar/clamav/0.99.2_1`

```
-rw-r--r--   1 vhaisaryanw  staff   3.1K Apr 22  2016 AUTHORS
-rw-r--r--   1 vhaisaryanw  staff    18K Apr 22  2016 COPYING
-rw-r--r--   1 vhaisaryanw  staff   749K Apr 22  2016 ChangeLog
-rw-r--r--   1 vhaisaryanw  staff   700B Apr 25 13:40 INSTALL_RECEIPT.json
-rw-r--r--   1 vhaisaryanw  staff   428B Apr 22  2016 NEWS
-rw-r--r--   1 vhaisaryanw  staff    99K Apr 22  2016 README
drwxr-xr-x  11 vhaisaryanw  staff   374B Apr 25 13:40 bin/
-rw-r--r--   1 vhaisaryanw  staff    59B Apr 25 13:42 clamd.conf
-rw-r--r--   1 vhaisaryanw  staff     0B Apr 25 13:42 clamd.sock
-rw-r--r--   1 vhaisaryanw  staff    35B Apr 25 13:43 freshclam.conf
drwxr-xr-x   3 vhaisaryanw  staff   102B Apr 25 13:39 include/
drwxr-xr-x   9 vhaisaryanw  staff   306B Apr 25 13:40 lib/
drwxr-xr-x   3 vhaisaryanw  staff   102B Apr 25 13:39 sbin/
drwxr-xr-x   4 vhaisaryanw  staff   136B Apr 25 13:40 share/
```

Like you, those files reside in `${homebrew_dir}/etc/clamav`.  Do I need to do something extra? And do the instructions need to change?